src/org/usfirst/frc/team694/robot/commands/LiftDownCommand.java

```diff
@@ -22,7 +22,7 @@ protected void initialize() {
 
     // Called repeatedly when this Command is scheduled to run
     protected void execute() {
-        Robot.lift.goDown();
+        Robot.lift.setLiftMotors(-1);
```

Comment:
Again, I'd highly suggest against letting the user set the speed of the lift.

It's either 100% or 0%. If we make the `setLiftMotors` method public, we'll give our user the ability to burn out the motor.

Bring back `goDown()` and `goUp()` and use those.